### PR TITLE
Docs: Edits to streamline Install OpenVINO Overview Page

### PR DESCRIPTION
**Pull request by EdjeElectronics**
This pull request has edits to the `install-openvino-overview.md` page to make it simpler for new users to decide which package to install. It also changes the "What's Next?" section to have links to example notebooks that users can try in their browser before installing OpenVINO.

### Details:
 - Edit `install-openvino-overview.md` to list 3 options for installing OpenVINO, with Installing OpenVINO Development Tools being the recommended option for new users

### Tickets:
 - NA

### CODE REVIEW

File: docs/install_guides/installing-openvino-overview.md [hunk 1]
```diff
@@ -6,53 +6,64 @@
    :maxdepth: 3
    :hidden:
    
-   OpenVINO Runtime <openvino_docs_install_guides_install_runtime>
+   OpenVINO Runtime 2022.1 <openvino_docs_install_guides_install_runtime>
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:9]: This change is for 2022.1 only and need be reverted.

File: docs/install_guides/installing-openvino-overview.md [hunk 2]
```diff
@@ -6,53 +6,64 @@
...
-  * Benchmark Tool
-  * Accuracy Checker and Annotation Converter
-  * Model Downloader and other Open Model Zoo tools
+**C++** <br>
+For developers working in C++, the core OpenVINO Runtime libraries must be installed separately. Then, OpenVINO Development Tools can be installed using requirements files or PyPI. See the [For C++ Developers](installing-model-dev-tools.md#for-c-developers) section of the Install OpenVINO Development Tools page for instructions.
 
+### Option 2. Install OpenVINO Runtime only
 
-### Choose Your Installation Method
+The OpenVINO Runtime may also be installed on its own without OpenVINO Development Tools. This is recommended for users who already have an optimized model and want to deploy it in an application that uses OpenVINO for inference on their device. To install OpenVINO Runtime only, follow the instructions on the [Install OpenVINO Runtime](installing-openvino-runtime.md) page.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:47]:
```suggestion
OpenVINO Runtime may also be installed on its own without OpenVINO Development Tools. This is recommended for users who already have an optimized model and want to deploy it in an application that uses OpenVINO for inference on their device. To install OpenVINO Runtime only, follow the instructions on the [Install OpenVINO Runtime](installing-openvino-runtime.md) page.
```

File: docs/install_guides/installing-openvino-overview.md (hunk 1 above)
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:9]:
```suggestion
   OpenVINO Runtime <openvino_docs_install_guides_install_runtime>
```

File: docs/install_guides/installing-openvino-overview.md [hunk 3]
```diff
@@ -7,52 +7,63 @@
    :hidden:
    
    OpenVINO Runtime <openvino_docs_install_guides_install_runtime>
+   OpenVINO Runtime 2022.1.1 <openvino_docs_install_guides_install_runtime_2022_1_1>
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:10]:
```suggestion
```

File: docs/install_guides/installing-openvino-overview.md [hunk 4]
```diff
@@ -13,46 +13,56 @@
...
+  - Accuracy Checker and Annotation Converter
+  - Model Downloader and other Open Model Zoo tools
 
-**If you have already finished developing your models and converting them to the OpenVINO model format, you can [install OpenVINO Runtime](installing-openvino-runtime.md) to deploy your applications on various devices**. OpenVINO Runtime contains a set of libraries for easy inference integration with your products.
+### Option 1. Install OpenVINO Runtime and OpenVINO Development Tools (recommended)
+The best way to get started with OpenVINO is to install OpenVINO Development Tools, which will also install the OpenVINO Runtime Python package as a dependency. Follow the instructions on the [Install OpenVINO Development Tools](installing-model-dev-tools.md) page to install it.
 
-**If you want to download models from [Open Model Zoo](../model_zoo.md), [convert your own models to OpenVINO IR](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md), or [optimize and tune pre-trained deep learning models](../optimization_guide/model_optimization_guide.md)**, [install OpenVINO Development Tools](installing-model-dev-tools.md), which provides the following tools:
+**Python** <br>
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:38]:
```suggestion
**Python**
```

File: docs/install_guides/installing-openvino-overview.md [hunk 5]
```diff
@@ -13,46 +13,56 @@
...
+For developers working in Python, OpenVINO Development Tools can easily be installed using PyPI. See the [For Python Developers](installing-model-dev-tools.md#for-python-developers) section of the Install OpenVINO Development Tools page for instructions.
 
-  * Model Optimizer
-  * Post-Training Optimization Tool
-  * Benchmark Tool
-  * Accuracy Checker and Annotation Converter
-  * Model Downloader and other Open Model Zoo tools
+**C++** <br>
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:41]:
```suggestion
**C++**
```

File: docs/install_guides/installing-openvino-overview.md [hunk 6]
```diff
@@ -13,46 +13,56 @@
...
+### Option 1. Install OpenVINO Runtime and OpenVINO Development Tools (recommended)
+The best way to get started with OpenVINO is to install OpenVINO Development Tools, which will also install the OpenVINO Runtime Python package as a dependency. Follow the instructions on the [Install OpenVINO Development Tools](installing-model-dev-tools.md) page to install it.
 
-**If you want to download models from [Open Model Zoo](../model_zoo.md), [convert your own models to OpenVINO IR](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md), or [optimize and tune pre-trained deep learning models](../optimization_guide/model_optimization_guide.md)**, [install OpenVINO Development Tools](installing-model-dev-tools.md), which provides the following tools:
+**Python** <br>
+For developers working in Python, OpenVINO Development Tools can easily be installed using PyPI. See the [For Python Developers](installing-model-dev-tools.md#for-python-developers) section of the Install OpenVINO Development Tools page for instructions.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:39]: we need to apply the same changes for this branch too.

File: docs/install_guides/installing-openvino-overview.md [hunk 7]
```diff
@@ -13,46 +13,56 @@
...
 
-  * Model Optimizer
-  * Post-Training Optimization Tool
-  * Benchmark Tool
-  * Accuracy Checker and Annotation Converter
-  * Model Downloader and other Open Model Zoo tools
+**C++** <br>
+For developers working in C++, the core OpenVINO Runtime libraries must be installed separately. Then, OpenVINO Development Tools can be installed using requirements files or PyPI. See the [For C++ Developers](installing-model-dev-tools.md#for-c-developers) section of the Install OpenVINO Development Tools page for instructions.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:42]: we need to apply the same changes for this branch too.

**Comment by EdjeElectronics** [comment]
Hi @xu-yuan1 , I implemented your suggested changes and also modified the links to use <a> tags.

**Comment by xu-yuan1** [comment]
Hi @EdjeElectronics , this would also require the same changes to `installing-model-dev-tools.md`.

**Comment by EdjeElectronics** [comment]
This pull request was merged on the master branch (see PR #13155). Closing this pull request.

**Comment by kblaszczak-intel** [comment]
Is this change not relevant to 22.2? If so, why are we limiting it to the master branch?

**Pull request closed without merging**